src/mbgl/tile/geometry_tile_worker.hpp

```diff
@@ -28,14 +27,16 @@ class GeometryTileWorker {
     GeometryTileWorker(ActorRef<GeometryTileWorker> self,
                        ActorRef<GeometryTile> parent,
                        OverscaledTileID,
+                       const std::string&,
                        const std::atomic<bool>&,
                        const MapMode,
-                       const float pixelRatio);
+                       const float pixelRatio,
+                       const bool showCollisionBoxes_);
     ~GeometryTileWorker();
 
     void setLayers(std::vector<Immutable<style::Layer::Impl>>, uint64_t correlationID);
     void setData(std::unique_ptr<const GeometryTileData>, uint64_t correlationID);
-    void setPlacementConfig(PlacementConfig, uint64_t correlationID);
```

Comment:
Yeah, I definitely want to simplify `GeometryTileWorker` by removing the two-phase loading and all the accompanying state transitions. I started in on making those changes and realized it would significantly increase the footprint of this PR, so I decided it makes sense to defer to a follow-up.

I'll add an explanatory comment for now.